xce/lib/XChemThread.py

```diff
@@ -3199,27 +3204,15 @@ def parse_file_system(self):
             autoDir = self.processedDir
 
         self.Logfile.insert("checking for new data processing results in " + autoDir)
+        path_list = sorted(glob.glob(os.path.join(autoDir, "*")))
```

Comment:
I'm curious, how long did the glob.glob take for how many files?

Reply:
Only 0.4s in my test, it's far from being the actual culprit, but doesn't help things